src/Sites/twitch.tv/twitch.tsx

```diff
@@ -204,7 +206,7 @@ export class TwitchPageScript {
 	whenUpperLayerRequestsThePageScriptStopsSendingChatLinesUpstream(): void {
 		ffzMode = true;
 
-		let sets = this.twitch.getAutocompleteHandler()?.providers[0].props.emotes;
+		let sets = this.twitch?.getAutocompleteHandler()?.providers[0].props.emotes;
```

Comment:
`this.twitch` will always be undefined, since the decorator is calling this method which doesn't pass the context properly. This doesn't seem like the correct place to put this